code/src/index.css

```diff
@@ -1,13 +1,294 @@
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+:root {
+  --fieldset-border-color: #000;
+  --h1-text-color: #fff;
+  --h1-bg-color: #000;
+  --border-color: #7c7b7b;
+  --box-shadow-color: #000;
+  --textarea-border-color: #c9c8c8;
+  --data-text-color: #d2d2d2;
+  --invalid-text-color: red;
+  --valid-text-color: #000;
+  --button-bg-color: #ffadad;
+  --heart-bg-color: #a4a4a4;
+  --refresh-bg-color: #000;
```

Comment:
Nice with variables, but is it making it easier? Some variables u don't use that much and is only white or black.. are u saving any time with having all colors here? 🤔 Have not thought about this before..

Reply:
Yeah, maybe for this project it was quite unnecessary to use so many variables. But if I want to change the design and the color, it's much easier to change here in the root instead of scrolling through all the css.